### PR TITLE
Added message arguments #3769

### PR DESCRIPTION
**Pull request by yashLadha**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by RohanVB** [comment]
@yashLadha the PR heading isn't an issue, the commit heading is (that's what gitmate-bot is flagging).
Look at http://api.coala.io/en/latest/Developers/Git_Basics.html for rebasing (note: you'll need to squash the extra commits); ask on gitter for further help :D

File: coalib/settings/Section.py [hunk 1]
```diff
@@ -209,8 +209,6 @@ def __getitem__(self, item, ignore_defaults=False):
                 res.value = self.defaults[key]._value + ', ' + res._value
                 res.to_append = False
                 return res
-            res.to_append = False
-            return res
```
Comment (RohanVB) [inline coalib/settings/Section.py]: Why was this deleted? It isn't related to #3769

**Comment by yashLadha** [comment]
See for commit it is in issue #3822

**Comment by RohanVB** [comment]
That's a separate issue @yashLadha.

**Comment by yashLadha** [comment]
I will take care in future. I am new to the community so i need to learn

**Comment by yash-nisar** [comment]
https://github.com/coala/coala/issues/3769 has already been taken by someone. You can still work on https://github.com/coala/coala/issues/3822 by assigning it to yourself. I thoroughly recommend you to go through http://api.coala.io/en/latest/Developers/Newcomers_Guide.html before working on anything. However I noticed that you have the dreaded merge commit which clutters our commit history. You need to go through http://api.coala.io/en/latest/Developers/Git_Basics.html too.

**Comment by jayvdb** [comment]
@yashLadha, please do not do development on your "master" branch.  Follow our newcomer guide.
